about/cascading-style-sheets.md

```diff
@@ -0,0 +1,91 @@
+#Reading CSS
+
+##Properties
+
+###Display
+
+####Block-level elements
+
+Block-level elements stretch from left to right as far as they can
+
+1. div: standard block-level element
+2. p
+3. form
+4. header
+5. footer
+6. section
+
+width: setting the width of a block-level element prevents it from stretching to the edges of its containers
+  * max-width: using max-width instead of width improves the browser's handling of small windows, important for making sites usable on mobile, supported by all browsers
+  * percent width: sets a unit relative to containing block
+    * Examples: used with images to set image 50% the size of containing element, can be used with min-width and max-width to limit how big or small the image can get
+margin: setting the margin to auto will horizontally center the element in its container
+
+####Inline
+
+Wraps text within a paragraph without disrupting paragraph flow
+
+1. span: standard inline-level element
+2. a: most common inline-level element, used for links
+
+####Inline-block
+
+inline-block: is used to create a grid of boxes that fills the browser width and wraps nicely
+  * Use hasLayout to support on old browsers
+  * Can be used for layouts
+    * vertical-align property: will most likely want to set to top
+    * Need to set the width of each column
+    * There will be a gap between the columns if there is any whitespace between them in the HTML
+
+####None
+
+Displays nothing, renders the page as though the element doesn't exist, used by javascript
+
+1. script: defaults to none
+
+####Position
```

Comment:
This should probably go back down a heading level, since it's a new property... BTW are CSS properties case-sensitive?